index.php

```diff
@@ -56,36 +97,40 @@
         <h2 class="promo__title">Нужен стафф для катки?</h2>
         <p class="promo__text">На нашем интернет-аукционе ты найдёшь самое эксклюзивное сноубордическое и горнолыжное снаряжение.</p>
         <ul class="promo__list">
-            <!--заполните этот список из массива категорий-->
-            <li class="promo__item promo__item--boards">
-                <a class="promo__link" href="pages/all-lots.html">Имя категории</a>
-            </li>
+            <!-- Show lots categories -->
+            <?php foreach ($categories as $category): ?>
+                <li class="promo__item promo__item--boards">
+                    <a class="promo__link" href="pages/all-lots.html"><?=$category;?></a>
```

Comment:
На будущее для читаемости можно сделать так:
```suggestion
                    <a class="promo__link" href="pages/all-lots.html">
                        <?= $category ?>
                    </a>
```

Reply:
Точка с запятой при вызове. <?= $category ?> необязательна